.gitignore

```diff
@@ -26,4 +26,4 @@
 .idea
 
 # Ignore database.yml file
-/config/database.yml
+# /config/database.yml
```

Comment:
I'd delete this and the previous line.  The "Ignore database.yml file" is no longer correct, and I don't see any advantage to leaving around unused lines.